google/ggrc-core
# GGRC-5504 Add 'read-more' for object description on global mapper modal

# Issue description

"Show more" link is not displayed in Global Search for description field of any object.

# Steps to test the changes

1. Add long description (more then 5 lines of text) for any object
2. Open Global Search and find the object from step#1
3. Click grey triangle to expand object info
4. Look at the Description

Description field contains 'Read More' link. After clicking the link the full description is shown.

# Solution description

'div' tag in appropriate template was replaced with 'read-more'.

# Sanity checklist




# PR Review checklist
